nextflow.config

```diff
@@ -52,6 +52,9 @@ params {
     min_val_gl                  = 0.9
     min_val_dp                  = 5
 
+    // Imputation
+    batch_size                 = 100
```

Comment:
From my point of you, I would keep it to 100 the aim is to have a pipeline running efficiently, because some user would just launch the pipeline without adding this parameter and this would take a huge amount of time to compute.
The errors trowed a quite explicit and would help the user to change the parameters accordingly.

> If this is 100 by default but the input samples are less than this, it won't work for some tools

No it should work perfectly fine with a lower number of individuals.